librosa/core/audio.py

```diff
@@ -481,23 +481,31 @@ def resample(
         target sampling rate
 
     res_type : str
-        resample type (see note)
+        resample type
+
+        'kaiser_best' (default)
+            `resampy` high-quality mode
+        'kaiser_fast'
+            `resampy` faster method
+        'fft' or 'scipy'
+            `scipy.signal.resample` Fourier method.
+        'polyphase'
+            `scipy.signal.resample_poly` polyphase filtering. (fast)
+        'linear'
+            `samplerate` linear interpolation. (very fast)
+        'zero_order_hold'
+            `samplerate` repeat the last value between samples. (very fast)
+        'sinc_best', 'sinc_medium' or 'sinc_fastest'
+            `samplerate` high-, medium-, and low-quality sinc interpolation.
+        'soxr_vhq', 'soxr_hq', 'soxr_mq' or 'soxr_lq'
+            `soxr` Very high-, High-, Medium-, Low-quality FFT-based bandlimited interpolation.
+            ``'soxr_hq'`` is default setting of `soxr` (fast)
+        'soxr_qq'
+            `soxr` Quick cubic interpolation (very fast)
 
         .. note::
-            By default, this uses `resampy`'s high-quality mode ('kaiser_best').
-
-            To use a faster method, set ``res_type='kaiser_fast'``.
-
-            To use `scipy.signal.resample`, set ``res_type='fft'`` or ``res_type='scipy'``. (slow)
-
-            To use `scipy.signal.resample_poly`, set ``res_type='polyphase'``. (fast)
-
-            To use `samplerate.converters.resample`, set any of the following:
-
-                - ``res_type='linear'``: linear interpolation (fast)
-                - ``res_type='zero_order_hold'``: repeat the last value between samples (very fast)
-                - ``res_type='sinc_best'``, ``'sinc_medium'``, or ``'sinc_fastest'``: for high-, medium-,
-                  and low-quality sinc interpolation
+            `samplerate` and `soxr` are not installed with `librosa`.
+            To use `samplerate` or `soxr`, they should be installed manually.
```

Comment:
put a link to installation instructions here?

Reply:
Seems to me like this would be best resolved by having a proper soxr documentation site which we can intersphinx-link to.